connection_test.go

```diff
@@ -765,3 +766,35 @@ func TestConnectTimeout(t *testing.T) {
 		close(testComplete)
 	})
 }
+
+func TestTosPriority(t *testing.T) {
+	ctx, cancel := NewContext(time.Second)
+	defer cancel()
+
+	opts1 := testutils.NewOpts().SetServiceName("s1").SetTosPriority(tos.LOWDELAY).NoRelay()
+	opts2 := testutils.NewOpts().SetServiceName("s2").NoRelay()
```

Comment:
`NoRelay` should only be required for options passed to `WithTestServer`, can remove it here